endpoints-framework/src/test/java/com/google/api/server/spi/dispatcher/PathDispatcherTest.java

```diff
@@ -121,4 +121,10 @@ public void builderNullHandler() {
       // expected
     }
   }
+
+  @Test
+  public void methodNotKnown() throws IOException {
+    PathDispatcher<DispatcherContext> dispatcher = PathDispatcher.builder().build();
+    assertThat(dispatcher.dispatch("HEAD", "test/one/two/3", context)).isFalse();
```

Comment:
isn't `HEAD` supposed to be legal on any url for which `GET` is legal? the RFC suggests that's the case (it uses SHOULD, though, not MUST)

perhaps a made-up method would be better here